cmd/minishift/cmd/start.go

```diff
@@ -144,7 +144,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	proxyConfig := handleProxies()
 
 	machineConfig := cluster.MachineConfig{
-		MinikubeISO:      viper.GetString(isoURL),
+		MinikubeISO:      viper.GetString(config.SetUrl.Name),
```

Comment:
@gbraad the way struct created. this is a sample not implemented completely, wanted to check with @hferentschik if this what he have in mind.